target/linux/ipq40xx/image/Makefile

```diff
@@ -221,6 +221,31 @@ define Device/qcom_ap-dk04.1-c1
 endef
 TARGET_DEVICES += qcom_ap-dk04.1-c1
 
+define Device/qxwlan_e2600ac-c1
+	$(call Device/FitImage)
+	DEVICE_TITLE :=  Qxwlan E2600AC-C1 (nor flash)
+	BOARD_NAME := e2600ac-c1
+	DEVICE_DTS := qcom-ipq4019-qxwlan-e2600ac-c1
+	KERNEL_SIZE := 4096k
+	IMAGE_SIZE := 31232k
+	IMAGES := sysupgrade.bin
+	IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | append-metadata
+	DEVICE_PACKAGES := ipq-wifi-qxwlan_e2600ac
```

Comment:
Please don't forget to add the changes to the package/firmware/ipq-wifi there. The ImageBuilder builts requires that all specified package are available.

Reply:
Hi,
    What should i do?

(https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/firmware/ipq-wifi/Makefile;h=39d29a4ff388ca42c1659bdc924cb2c883a4b729;hb=87c42101cfb001b4bd418d1201fa4d8c822dc77b)